builders/substrate/libraries/py-substrate-interface.md

```diff
@@ -303,7 +304,7 @@ except SubstrateRequestException as e:
 
 ### Offline Signing {: #offline-signing }
 
-You can sign transaction payloads or any arbitrary data using a keypair object through the [`sign`](https://polkascan.github.io/py-substrate-interface/#substrateinterface.Keypair.sign){target=\_blank} method. This can be used for offline signing of transactions.
+You can sign transaction payloads or any arbitrary data using a keypair object through the [`sign`](https://jamdottech.github.io/py-polkadot-sdk/usage/keypair-creation-and-signing/#keypair-creation-and-signing){target=\_blank} method. This can be used for offline signing of transactions.
```

Comment:
```suggestion
You can sign transaction payloads or any arbitrary data using a keypair object through the [`sign`](https://jamdottech.github.io/py-polkadot-sdk/reference/keypair/#substrateinterface.keypair.Keypair.sign){target=\_blank} method. This can be used for offline signing of transactions.
```